tests/test_onnxml_normalizer_converter.py

```diff
@@ -0,0 +1,94 @@
+"""
+Tests onnxml Normalizer converter
+"""
+import unittest
+import warnings
+
+import numpy as np
+import torch
+from sklearn.preprocessing import Normalizer
+
+from hummingbird.ml._utils import onnx_ml_tools_installed, onnx_runtime_installed, lightgbm_installed
+from hummingbird.ml import convert
+
+if onnx_runtime_installed():
+    import onnxruntime as ort
+if onnx_ml_tools_installed():
+    from onnxmltools import convert_sklearn
+    from onnxmltools.convert.common.data_types import FloatTensorType as FloatTensorType_onnx
+
+
+class TestSklearnNormalizer(unittest.TestCase):
+    def _test_normalizer_converter(self, norm):
+        warnings.filterwarnings("ignore")
+        X = np.array([[1, 2, 3], [4, 3, 0], [0, 1, 4], [0, 5, 6]], dtype=np.float32)
+
+        # Create SKL model for testing
+        model = Normalizer(norm=norm)
+        model.fit(X)
+
+        # Create ONNX-ML model
+        onnx_ml_model = convert_sklearn(model, initial_types=[("float_input", FloatTensorType_onnx(X.shape))])
+
+        # Create ONNX model by calling converter
+        onnx_model = convert(onnx_ml_model, "onnx", X)
+        # Get the predictions for the ONNX-ML model
+        session = ort.InferenceSession(onnx_ml_model.SerializeToString())
+        output_names = [session.get_outputs()[i].name for i in range(len(session.get_outputs()))]
+        onnx_ml_pred = [[] for i in range(len(output_names))]
+        inputs = {session.get_inputs()[0].name: X}
+        pred = session.run(output_names, inputs)
+        for i in range(len(output_names)):
```

Comment:
I don't think this is actually needed. I think only one `output_name` exists. Can you please check?